### PR TITLE
Use the theme's regular background color(in Dark Theme)

### LINKED ISSUE #6785: Dark Theme Inconsistency between main page and mails
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6-400

### Where did you get the app from?

Google Play

### Android version

13

### Device model

_No response_

### Steps to reproduce

Chose dark theme. 

### Expected behavior

The same background colour everywhere. 

### Actual behavior

Main background is more grey, while mails view is dark black (Amoled). However, in the settings both settings are the same "dark mode". 

### Logs

_No response_

### PR DESCRIPTION
**Pull request by mhf1998**
Fixes #6785 

In the mentioned issue, a problem was reported that the background color was different in the message display and message list pages. By talking more in that issue, we ended up making some changes to achieve the goals you [mentioned in that issue](https://github.com/thundernest/k-9/issues/6785#issuecomment-1598584382):

> The goal is to use the theme's regular background color.
> What needs to be done:
> - Remove the theme attribute messageViewBackgroundColor and all code associated with it.
> - Change DisplayHtml.cssStyleTheme() and MessageWebView.configure() to set the WebView's background color to the theme background color, not black."

In this branch, the changes were applied and we can see the background color difference before and after these changes:

|Before|After|
|-|-|
|message_view_Before|message_view_After|
|compose_reply(html type)_Before|compose_reply(html type)_After|

### CODE REVIEW

File: app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java [hunk 1]
```diff
@@ -271,7 +271,6 @@ public void onCreate(Bundle savedInstanceState) {
         // background color needs to be forced
         //TODO: Change themes to use appropriate background colors that don't need overriding.
         TypedValue outValue = new TypedValue();
-        themeContext.getTheme().resolveAttribute(R.attr.messageViewBackgroundColor, outValue, true);
```
Comment (cketti) [inline app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java:274]: This only removes half of the code belonging to the feature. We end up with outdated comments and broken code (a background color is set, it just happens to accidentally be Color.TRANSPARENT).

File: app/ui/legacy/src/main/java/com/fsck/k9/view/MessageWebView.kt [hunk 2]
```diff
@@ -34,7 +34,7 @@ class MessageWebView : WebView, KoinComponent {
         isLongClickable = true
 
         if (config.useDarkMode) {
-            setBackgroundColor(0xff000000L.toInt())
+            setBackgroundColor(android.R.attr.windowBackground)
```
Comment (cketti) [inline app/ui/legacy/src/main/java/com/fsck/k9/view/MessageWebView.kt:37]: `setBackgroundColor()` takes a color as argument, not a resource attribute ID.

**Review by cketti — CHANGES_REQUESTED** [review]

File: app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java (hunk 1 above)
Comment (mhf1998) [reply app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java:274]: yes. I fixed this, in new commits.

**Comment by cketti** [comment]
Thanks for working on this :+1: I picked up your work and modified it slightly in #7199. Closing this pull request.

**Comment by mhf1998** [comment]
> Thanks for working on this 👍 I picked up your work and modified it slightly in #7199. Closing this pull request.

Hello. You're welcome. 🙏🏻☺️
Thanks.

**Pull request closed without merging**